lib/datadog/lambda.rb

```diff
@@ -35,9 +36,11 @@ def self.configure_apm
       $stdout.sync = true
 
       Datadog.configure do |c|
-        c.tracing.writer = Datadog::Tracing::SyncWriter.new(
-          transport: Datadog::Transport::IO.default
-        )
+        unless Datadog::Utils.extension_running
```

Comment:
Should we add a test scenario to cover this change?

Reply:
We have an internal self-monitoring app to test this, since we can't intercept them here. For all the Datadog Lambda Layers we flush to logs to test this. 

I've manually tested it, this tests will be added to the self-monitoring app once this is deployed!